GCTC-NTGC/gc-digital-talent
# Feature - 2790 - Add New Search Container/Form

Resolves #2790 

## Summary

This duplicates the existing `SearchContainer` and `SearchForm` putting them behind the feature flag `FEATURE_APPLICANTSEARCH`.

## Testing

1. Rebuild your php container
2. Build the `talentsearch` frontend
3. Navigate to `/talent/search`
4. Test the form (it should function no differently)
